base_layer/wallet/src/util/emoji.rs

```diff
@@ -77,6 +77,11 @@ lazy_static! {
 #[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
 pub struct EmojiId(String);
 
+/// Returns the current emoji set as a vector of char
+pub fn emoji_set() -> Vec<char> {
+    EMOJI.to_vec()
+}
+
```

Comment:
No need to copy the emojis into memory first

```suggestion
/// Returns the emoji set 
pub const fn emoji_set() -> [char;256] {
    EMOJI
}

```